addons/better-quoter/addon.json

```diff
@@ -22,8 +22,8 @@
   "info": [
     {
       "type": "notice",
-      "text": "Scratchblocks tags are not supported, and the bbcode may not be 100% accurate. To get fully accurate quotes, quote as normal and delete any parts that you don't need.",
-      "id": "scratchblocks_notice"
+      "text": "The bbcode may not be 100% accurate. To get fully accurate quotes, quote as normal and delete any parts that you don't need.",
+      "id": "bbcode_inaccuracy"
```

Comment:
I know this was already wrong, but it should be `BBCode` instead of `bbcode`.
Also, keep the ID so that translators can change the string more easily.